src/strategy-adapter.ts

```diff
@@ -37,8 +38,7 @@ export class StrategyAdapter<T> {
     return new Promise<T | void>((resolve, reject) => {
       // mix-in passport additions like req.logIn and req.logOut
       for (const key in passportRequestMixin) {
-        /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-        (request as any)[key] = passportRequestMixin[key];
+        (request as AnyObject)[key] = passportRequestMixin[key];
```

Comment:
why any object here

Reply:
bcz we are adding more keys (logIn, login, logOut, logout, isAuthenticated, isUnauthenticated) in request of type Request (@loopback/repository). I am not completely sure, if any other keys were being added. I just console.log(key) in loop. That's why, I defined AnyObject